modules/apps/foundation/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectSessionImpl.java

```diff
@@ -14,20 +14,21 @@
 
 package com.liferay.portal.security.sso.openid.connect.internal;
 
+import java.io.Serializable;
+
+import com.liferay.portal.security.sso.openid.connect.OpenIdConnectSession;
 import com.nimbusds.oauth2.sdk.id.State;
 import com.nimbusds.oauth2.sdk.token.AccessToken;
 import com.nimbusds.oauth2.sdk.token.RefreshToken;
 import com.nimbusds.openid.connect.sdk.Nonce;
 import com.nimbusds.openid.connect.sdk.claims.UserInfo;
 
-import java.io.Serializable;
-
 /**
  * @author Edward C. Han
  */
-public class OpenIdConnectSession implements Serializable {
+public class OpenIdConnectSessionImpl implements OpenIdConnectSession, Serializable {
```

Comment:
I would rename this to NimbusDSOpenIdConnectSessionImpl since this is tied specifically to that